wire/wire.go

```diff
@@ -13,16 +14,18 @@ const (
 
 // Enumerate all valid MsgType values.
 const (
-	MsgTypePush = uint16(1)
-	MsgTypePull = uint16(2)
-	MsgTypeSync = uint16(3)
+	MsgTypePush    = uint16(1)
+	MsgTypePull    = uint16(2)
+	MsgTypeSync    = uint16(3)
+	MsgTypeDirect = uint16(4)
 )
 
 // Msg defines the low-level message structure that is sent on-the-wire between
 // peers.
 type Msg struct {
 	Version uint16 `json:"version"`
 	Type    uint16 `json:"type"`
+	To 		id.Signatory `json:"to"`
```

Comment:
Formatting is wrong. Also, is it appropriate to use `id.Signatory` as the type? What if this is a subnet identifiers? All signatories are hashes, not all hashes are signatories.